src/celeritas/CMakeLists.txt

```diff
@@ -78,6 +78,9 @@ list(APPEND SOURCES
   user/StepCollector.cc
 )
 
+# Bounding box used for output is in "orange"
+list(APPEND PUBLIC_DEPS Celeritas::orange)
```

Comment:
This implies that `ORANGE` is always built, right?

Reply:
But it sounds like `bounding box` should be in a library that both `Orange` and the `output` depends on.

Reply:
Yes, there are "core geometry " components in orange, such as types and interfaces. Splitting them out to another library is too far at this point. Better might be to move the other two wrappers into this one and rename the library "celgeo" or something

Reply:
Alright.  This is not strictly related to this PR so moving the discussion off-line.